### PR TITLE
Add a slider for the padding setting in the SUI

### PR DESCRIPTION
**Pull request by PankajBhojwani**
Replaces the textbox used for the padding setting with a slider

References #8764

### CODE REVIEW

File: src/cascadia/TerminalSettingsEditor/PaddingConverter.cpp [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+                    maxVal = curVal;
+                }
+            }
+        }
+        catch (...)
+        {
+            // If something goes wrong, even if due to a single bad padding value, we'll return default 0 padding
+            maxVal = 0;
+            LOG_CAUGHT_EXCEPTION();
+        }
```
Comment (PankajBhojwani) [inline src/cascadia/TerminalSettingsEditor/PaddingConverter.cpp:52]: This is basically what `TermControl` does to parse the padding (with a few edits made). Thoughts on whether this should be deduplicated somehow?

**Comment by mdtauk** [comment]
This would require going into the JSON file to set different padding values for the 4 edges right?

**Comment by DHowett** [comment]
@mdtauk yes.

**Comment by mdtauk** [comment]
> @mdtauk yes.

Unfortunate, but a slider is much simpler a UI - how will this be displayed, if I have set my own values.  Would there be some kind of textblock indicating the custom values that have been set?

**Comment by DHowett** [comment]
Users who want complicated things _during the 1.6 preview_ are free to use the JSON file, or the simplified UI that will replace their complicated setting with a simpler one.

**Comment by mdtauk** [comment]
> Users who want complicated things _during the 1.6 preview_ are free to use the JSON file, or the simplified UI that will replace their complicated setting with a simpler one.

Well that's me put in my place then lol
image

**Comment by DHowett** [comment]
Fair. :smile: There was a discussion about this in #8764. Mind joining in over there?

File: src/cascadia/TerminalSettingsEditor/PaddingConverter.cpp (hunk 1 above)
Comment (zadjii-msft) [reply src/cascadia/TerminalSettingsEditor/PaddingConverter.cpp:52]: IMO, ain't nobody got time for that right now.

**Review by zadjii-msft — APPROVED** [review]
Yea I'm cool with this for 1.6

File: src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw [hunk 2]
```diff
@@ -557,7 +557,7 @@
   <data name="Profile_IconBrowse.Content" xml:space="preserve">
     <value>Browse...</value>
   </data>
-  <data name="Profile_Padding.Header" xml:space="preserve">
+  <data name="Profile_Padding.Text" xml:space="preserve">
```
Comment (DHowett) [inline src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw:560]: We can't take this loc change -1 days before we ship. You're gonna have to figure out how to make up for it in code. Keep it `Header` and use `RS_`, file a bug to remove the workaround.

**Review by DHowett — CHANGES_REQUESTED** [review]

File: src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw (hunk 2 above)
Comment (PankajBhojwani) [reply src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw]: Fixed!

**Review by DHowett — APPROVED** [review]
Good for 1.6.

**Comment by DHowett** [comment]
padding

**Pull request merged**